src/less/src/common.less

```diff
@@ -79,6 +79,8 @@ input:not([type=button]):not([type=submit]).w2ui-input, textarea.w2ui-input {
     background-color: @input-background-color;
     line-height: normal;
     &:focus {
+        outline-style: solid;
+        outline-width: 1px;
```

Comment:
Chrome. 

I looked up and it seems that in FF it has nice focus border, not 1px though. Plus this code will break nice border in chrome, which is with a blur.